tools/make/tests.toml

```diff
@@ -64,22 +63,26 @@ set_env CARGO_TARGET_DIR ${project_dir}/target
 
 exec --fail-on-error cargo build -p icu_datagen
 
-cd docs/tutorials/testing
+cd docs/tutorials
 
-exec --fail-on-error cargo --config patch.toml test -p tutorials-test
-exec --fail-on-error cargo --config patch.toml run -p tutorial_baked
-exec --fail-on-error cargo --config patch.toml run -p tutorial_experimental
-exec --fail-on-error cargo --config patch.toml run -p tutorial_lazy_static
-exec --fail-on-error cargo --config patch.toml run -p tutorial_testdata
+mkdir crates/bin
+cp ${project_dir}/target/debug/icu4x-datagen crates/bin/icu4x-datagen
+
+exec --fail-on-error cargo --config testing/patch.toml test -p tutorials-test
+exec --fail-on-error cargo --config testing/patch.toml run -p tutorial_default
+exec --fail-on-error cargo --config testing/patch.toml run -p tutorial_experimental
+exec --fail-on-error cargo --config testing/patch.toml run -p tutorial_lazy_static
 
-# Build the postcard file
 # Copy our own icu4x-datagen and tutorial_buffer to speed up the task
```

Comment:
Issue: you're no longer copying icu4x-datagen as this comment suggests that we do